dali/operators/reader/loader/loader_test.cc

```diff
@@ -54,8 +57,88 @@ TYPED_TEST(DataLoadStoreTest, LMDBTest) {
     // shared_ptr to sample is destroyed
     auto sample = reader->ReadOne(false);
   }
+}
 
-  return;
+TYPED_TEST(DataLoadStoreTest, FileLabelLoaderMmmap) {
+  for (bool dont_use_mmap : std::vector<bool>{true, false}) {
```

Comment:
```suggestion
  for (bool dont_use_mmap : { true, false}) {
```
should suffice - it will iterate an initializer list

Reply:
Done